editor/src/components/inspector/common/inspector-end-to-end-tests.spec.browser2.tsx

```diff
@@ -3256,10 +3250,10 @@ describe('inspector tests with real metadata', () => {
 
       const elementFrame = getFrame(targetPath, renderResult)
 
-      await setControlValue('position-left-number-input', '', renderResult.renderedDOM)
+      await setControlValue('frame-left-number-input', '', renderResult.renderedDOM)
 
+      // Nothing changed
       expect(getFrame(targetPath, renderResult)).toBe(elementFrame)
```

Comment:
I kept these tests around to ensure that nothing changed is the result, but the bigger context is that the number control doesn't allow you to enter empty value or % value anymore, so there's no need to fire a toast.